packages/fury-adapter-oas3-parser/lib/parser/oas/parseMediaTypeObject.js

```diff
@@ -96,7 +96,7 @@ function parseMediaTypeObject(context, MessageBodyClass, element) {
       }
 
       if (examples.length > 1) {
-        parseResult.push(createWarning(namespace, `'${name}' 'examples' only one example is supported, other examples have been ignored`, examples));
+        parseResult.push(createWarning(namespace, `'${name}' 'examples' only one example is supported, other examples have been ignored`, examples.content[1]));
```

Comment:
"examples" is an array and thus doesn't have source maps, we should point to the 1st example which causes the warning for source maps.

Reply:
Isn't this an ideal case for providing multiple source map ranges? 😉 Not that I need them, this solution is fine for Dredd.